### DISCUSSION TITLE
Strange button/input mapping

### DISCUSSION THREAD

**Discussion by SomeGuy1916** [nefarius/DsHidMini#306] [Support]
Hello.

I am having trouble with the DsHidMini inputs. 

Depending on the device mode I chose (Xinput, SDF etc...) certain parts of the controller (buttons, triggers, dpad, analogue stick) either will not work, or will be mapped to the wrong location i.e. when I press button1 it might register as an input from button5. some buttons register as permanently activated, as though I am continually holding them down. Each device mode has a particular set of such errors that manifest the same way each time that device mode is chosen.  

I've also noticed that when in GPJ (generic direct input) mode, gamepad-tester.com will register inputs for both a player 1 and player 2 despite me only having one controller attached.

Additionally DS4windows mode registers no inputs whatsoever. 

The controller I am using is not aftermarket and has been used successfully in the past. I haven't used scp toolkit or other ps3 controller drivers on this computer before. I've tried uninstalling and re-installing DsHidMini, and I've double checked to make sure I've installed the drivers from the x64 folder to match my x64 based system. 

Any help or advice would be appreciated. 
Thank you.

**Comment by nefarius** [comment]
Every Mode has been tested and works. No one mode can magically work for all games and tools out there, you need to configure it as pointed out multiple times in the documentation, figure it out and it will work. Good luck.
